xonsh/history.py

```diff
@@ -673,6 +653,9 @@ def _hist_parse_args(args):
     parser = _hist_create_parser()
     if not args:
         args = ['show']
+    elif args[0] in _HIST_SESSIONS:
+        print('Use the `show` subcommand to get history sessions.', file=sys.stderr)
+        raise SystemExit
```

Comment:
Don't raise SystemExit or call sys.exit(), please.  This can/will take out the whole xonsh interpreter.